.changeset/afraid-dryers-happen.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/fiori-elements-writer': minor
+'@sap-ux/ui5-info': minor
```

Comment:
ui5-config should have a entry here too

Reply:
Hmm, I removed 2 unused dev dependencies so there is no practical difference in the delivered module from npmjs so I think its not necessary?